### PR TITLE
release: Bumped to 1.8.0

### PR DESCRIPTION
**Pull request by parkhomenko**
Bumped to the new 1.8.0 version

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 1.8.2
```
Comment (faunaee) [inline CHANGELOG.md:3]: The last release was 1.7.2. Shouldn't this be 1.8.0? Or, perhaps, 1.7.3?

**Review by faunaee — COMMENTED** [review]
Question about the chosen version.

File: CHANGELOG.md (hunk 1 above)
Comment (parkhomenko) [reply CHANGELOG.md:3]: oh, yes, sure, it's `1.8.0` thanks Ewan

Comment (parkhomenko) [reply CHANGELOG.md]: fixed

**Review by faunaee — APPROVED** [review]
LGTM! 👍

**Pull request merged**